package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "build": "webpack --mode=production",
+    "build": "rm -rf ./dist && webpack --mode=production",
```

Comment:
looks like there is no need for clearing dist folder, because it is already done by config

https://github.com/shpp/programming.org.ua/blob/master/webpack.config.js#L87

https://webpack.js.org/guides/output-management/#cleaning-up-the-dist-folder

Reply:
thanks, fixed